data/processSynapseJSON.ts

```diff
@@ -409,7 +425,10 @@ function processSynapseJSON(
     // split entity into 2 separate entities, Diagnosis and Demographics, if component is 'SRRSClinicalDataTier2'
     flatData = _.flatten(
         flatData.map((d) => {
-            if (d.Component === 'SRRSClinicalDataTier2') {
+            if (
+                d.Component === 'SRRSClinicalDataTier2' ||
+                d.Component === 'Therapy'
```

Comment:
This case is specific to `SRRSClinicalDataTier2` where we split the data into two separate entities (Diagnosis and Demographics). This will most likely cause some side effects. I think we should handle `Therapy` in the `getSampleAndPatientData` and attach a new `theraphy` object there.